### DISCUSSION TITLE
[Feature] People face picture selection

### DISCUSSION THREAD

**Discussion by weber8thomas** [immich-app/immich#2756] [Feature Request]
### The feature

Hi immich community,

Many thanks for the incredible work here, stunning!This is exactly what I was looking for for a long time!

Would it be possible to select/change the picture set by default to people face identified by ML?

Many thanks, 

Thomas 

### Platform

- [ ] Server
- [X] Web
- [ ] Mobile

**Comment by alextran1502** [comment]
Hello, you can follow this discussion for future update of FR feature https://github.com/immich-app/immich/discussions/2472

**Reply by weber8thomas** [reply]
Great! :) Many thanks
